src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -140,20 +137,17 @@ private void processSchema(Document doc, Element sequenceElement, String name, S
                     if (schema.valueSchema().isOptional()) {
                         schemaElement.setAttribute("minOccurs", "0");
                     }
-                }
-                else {
+                } else {
```

Comment:
```suggestion
                } 
                else {
```

Unrelated to this specific PR